imagetest/test_suites/guestagent/telemetry_test.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2023 Google LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     https://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build cit
+// +build cit
+
+package guestagent
+
+import (
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	daisyCompute "github.com/GoogleCloudPlatform/compute-daisy/compute"
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+func restartAgent(t *testing.T) {
+	var cmd *exec.Cmd
+	if utils.IsWindows() {
+		cmd = exec.Command("powershell.exe", "-NonInteractive", "Restart-Service", "GCEGuestAgent")
+	} else {
+		cmd = exec.Command("systemctl", "restart", "google-guest-agent")
+	}
+	err := cmd.Run()
+	if err != nil {
+		t.Fatalf("could not restart agent: %v", err)
+	}
+}
+
+func getAgentOutput(t *testing.T) string {
+	var cmd *exec.Cmd
+	if utils.IsWindows() {
+		cmd = exec.Command("powershell.exe", "-NonInteractive", "Get-WinEvent", "-Providername", "GCEGuestAgent")
+	} else {
+		cmd = exec.Command("journalctl", "-o", "cat", "-eu", "google-guest-agent")
+	}
+	out, err := cmd.Output()
+	if err != nil {
+		t.Fatalf("could not get agent output: %v", err)
+	}
+	return string(out)
+}
+
+func TestTelemetryEnabled(t *testing.T) {
+	output := getAgentOutput(t)
+	if !strings.Contains(output, "Successfully scheduled job telemetryJobID") {
```

Comment:
side note: in the future we can leverage the command monitor to tell the status of subsystems.

Reply:
That will be good. Much more reliable than this, I fear it will be flaky. It works for now at least.